src/category/mapper/get-categories.mapper.ts

```diff
@@ -0,0 +1,9 @@
+import { MenuItemCategory } from '@prisma/client';
+
+export class GetCategoriesMapper {
```

Comment:
the naming is not good. Mapper is something that takes one thing as param and return mapped item as response. So CategoryMapper sounds more logical, it takes category as param and returns mapped category (for ui).